sbws/config.log.default.ini

```diff
@@ -27,20 +27,18 @@ class = handlers.RotatingFileHandler
 formatter = to_file
 args = ('/dev/null', )
 
-## for logging to system log
-#[handler_to_syslog]
-#class=handlers.SysLogHandler
-#formatter=to_syslog
-#level = INFO
-#args = ()
+# for logging to system log
+[handler_to_syslog]
+class=handlers.SysLogHandler
+formatter=to_syslog
+level = INFO
```

Comment:
Please don't set the level here so it is more straightforward to understand what messages will (not) be ignored.

Reply:
ok, as commented above